Wrong version of peerDependency for rx-recompose

`ERROR in ./~/rx-recompose/observeProps.js
Module not found: Error: Cannot resolve module 'recompose/createHelper' `

`npm ERR! peerinvalid Peer rx-recompose@0.1.2 wants recompose@^0.8.0`

`createHelper` is only exists in 0.9.0 onwards.

--- a/src/packages/rx-recompose/package.json
+++ b/src/packages/rx-recompose/package.json
@@ -18,7 +18,7 @@
     "lodash": "^3.0.0"
   },
   "peerDependencies": {
-    "recompose": "^0.8.0",
+    "recompose": "^0.9.1",
     "react": "^0.14.0",
     "rx": "^4.0.0"
   },

